### DISCUSSION TITLE
Approved application as a teacher but no ability to upgrade organizations to free team plans

### DISCUSSION THREAD

**Discussion by johanneskoester** [community/community#50702] [GitHub Education]
I have successfully renewed my accounts approval as a teacher (@johanneskoester). However, I am unable to reactivate the free team plans of any of my organizations. Whenever I try that, I just see the normal billing form. Also, I cannot see the team plan under the benefit page. Please let me know what can be done.

**Comment by samus-aran** [comment]
Hi @johanneskoester 

Please visit your [GitHub Global Campus dashboard](https://education.github.com/globalcampus/teacher). You'll find the section "Upgrade your academic organizations" near the top of the page if you have organizations you can upgrade.

![Animation - scroll through Global Campus. Click the GitHub link "Upgrade to GitHub Team." Then click the Upgrade button next to the organizations you would like to upgrade](https://user-images.githubusercontent.com/10368374/168666981-4e726973-3e0f-41c2-aec0-7c9181c41830.gif)

Click the **Upgrade to GitHub Team** button and then click the **Upgrade** button next to the organization name to immediately upgrade it for free to GitHub Team.

Hope this helps! Let us know how you get along 👍

**Reply by jasonjan2003** [reply]
I've upgraded my organization from my Global Campus dashboard, but the org is still under a free plan...

**Reply by mihai-sysbio** [reply]
I can't update my organization https://github.com/MetabolicAtlas either, same issue

**Comment by cmatKhan** [comment]
I am having the same issue.

I noticed #55276 had a similar issue. The resolution was 'just to wait'. I was approved as an educator over a month ago, so I shouldn't be waiting on that at this point. 

But, maybe there is a lag between clicking the "upgrade" button, as shown above, and the upgrade appearing in the billing info?

**Reply by cmatKhan** [reply]
It depends on how you define 'resolved'.

It took months for our original request to be approved. It eventually was, and we'll be using codespaces for the class this year. But, last year, we were not able to due to the delay.

From the number of similar posts on this issue, it seems like others experience the same thing -- the long wait time seems unresolved. But it will eventually go through.
